VATSpy.dat

```diff
@@ -250,7 +250,7 @@ Slovakia|LZ|Radar
 Slovenia|LJ|Control
 Solomon Islands|AG|
 Somalia|HC|Radio
-South Africa|FA|Control
+South Africa|FA|Area

```

Comment:
This aswell..

This will make example FACA_CTR appear as "Cape Town Area Area" instead of "Cape Town Control"

Reply:
Got to say - it seemed like such a strange callsign for me 😅 

We generally do not include callsigns in the FIR name directly, as its handled by this line above.
This will now cause FACA_CTR to appear with "area" twice as its defined both here and in the FIR.

It will be better to remove the area from the FIR names, as "Cape Town FIR" is an offical FIR, and when atc is online, vatspy will automatically change it to "Cape Town Area"